client/navigation/stylesheets/index.scss

```diff
@@ -17,28 +17,53 @@
 		display: none !important;
 	}
 
+	#woocommerce-embedded-navigation {
+		position: fixed;
+		top: 0;
+		left: 0;
+		width: $navigation-width;
+		height: 100%;
+		box-sizing: border-box;
+		background-color: $gray-900;
+		z-index: 1100;
```

Comment:
Out of curiosity, what was the reason behind picking `1100`?  If it's higher than another, a comment explaining why it should be > n may be helpful to someone working on this in the future.